README.md

```diff
@@ -49,19 +50,32 @@ sudo apt-get install -y mongodb-org
 mongod --version
 ```
 
-A commonly missed step is not creating a directory for mongo to write to (this step my not be listed in other instructions because the directory is created during installation on Mac OS and Linux). 
+##### For Windows Users
+Download and install MongoDB Community Server, Mongo Shell, and Mongo Tools.
+https://www.mongodb.com/try/download/community
+https://www.mongodb.com/try/download/shell
+https://www.mongodb.com/try/download/database-tools
+
+Install Windows MongoDB Community Edition. “Install MongoD as a Service” and keep the default “Run service as Network Service user”. There is no need to install MongoDB Compass.
+
+#### Run a Mongo instance:
+##### For Mac Users
+Run a Mongo instance:
 
 ```
 mkdir -p ~/data/db
 sudo chown -R `id -un` ~/data/db
+sudo mongod --dbpath ~/data/db
 ```
 
-Run a Mongo instance: 
+##### For Windows Users
+Run a Mongo instance:
 
 ```
-sudo mongod --dbpath ~/data/db
+mongosh mongodb://localhost:27017/pa-wildflower-selector
```

Comment:
This doesn't run an instance of mongodb, you already did that by setting it up to "install mongod as a service." This opens the mongo shell, which is nice but not required for anything we do below I think. You could just say "for Windows, this happened automatically when you installed mongod as a service."